Formula/ocrmypdf.rb

```diff
@@ -3,9 +3,9 @@ class Ocrmypdf < Formula
 
   desc "Adds an OCR text layer to scanned PDF files"
   homepage "https://github.com/jbarlow83/OCRmyPDF"
-  url "https://files.pythonhosted.org/packages/24/e5/5f534c784afdcec76f8828aff1ad2e908929a85973188d0e833138f0f335/ocrmypdf-10.3.3.tar.gz"
-  sha256 "440c24b909f12991b7b8fb2ab56c82d200d8d93ece1010412296e8788bb4a624"
-  license "GPL-3.0"
```

Comment:
interesting, they actually relicense the code recently, https://github.com/jbarlow83/OCRmyPDF/issues/600